src/Cmf/MainBundle/Resources/data/posts/2015-04-01.yml

```diff
@@ -0,0 +1,62 @@
+name: "cmf-hackday-may-2015"
+parent: "/news"
+title: "CMF Hackday 1st/2nd May 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-04-01"
+body: |
+  We are happy to announce a joint Symfony CMF / Sulu.io hackday in Dornbirn on the 1st and 2nd of May. Join us
+  for a hacking, discussion and workshops (if there is interest).
+
+  If you would like to come, please register [here](<http register here dot com>)
+
+  ### Schedule
+  
+  #### Friday 1st May
+
+  09:00 - Breakfast
+  09:30 - <workshop> / <talk> / <discussion> / <hacking>
+  12:00 - Lunch
+  12:30 - <workshop> / <talk> / <discussion> / <hacking>
+  18:00 - Mountain dinner at the [Karren](http://www.karren.at/?id=1800)
+
+  #### Saturday 2nd May
+
+  09:00 - Breakfast
+  09:30 - <workshop> / <talk> / <discussion> / <hacking>
+  12:00 - Lunch
+  12:30 - <workshop> / <talk> / <discussion> / <hacking>
+
```

Comment:
Any ideas for scheduled things? Some examples:

- Symfony CMF 2 discussion
- Jackalope 2 presentation (providing I can invest some time in it)
- Symfony CMF + Sulu - how Sulu is using the CMF
- Sulu workshop
- CMF workshop

I think the above things would depend on who expresses interest in coming - I think otherwise the focus can be on non-workshops.